### DISCUSSION TITLE
Power off?

### DISCUSSION THREAD

**Discussion by jtolio** [wasp-os/wasp-os#500] [General]
This is a weird question - I can't figure out how to turn a device with WaspOS off? Am I dumb?

I went through all the menus, tried all kinds of different swipes, pressed the hardware button, held it down, etc. 

My goal is to place the device in a box on a shelf for a potentially long period of time, after carefully making sure the lithium ion battery inside is in a healthy neutral range. So I don't want to run the battery out to kill it.

What do I do here?

**Comment by thiswillbeyourgithub** [comment]
Hi !

There's no power off on the pinetime, it's not related to wasp os, it's the device itself. It allows saving costs on power management. I stored it for like a year with 0 battery and it's still fine btw.

**Comment by Letsric** [comment]
I guess you are talking about the PineTime.

In this case, unfortunately, I think there is no way of turning the device off in general. (See [this reply over at Infinitime](https://github.com/InfiniTimeOrg/InfiniTime/issues/201#issuecomment-774421962)) You can disable Bluetooth using the DisaBLE app to save some power, but I believe that's as far as you can go without desoldering the battery. Speaking of which, this is actually a good option for storing it away, **if** you have a PineTime-Devkit. Refer to [the official Documentation](https://pine64.org/documentation/PineTime/Further_information/Devkit_wiring/) for this.

Personally, I stored the PineTime for about half a year at 0% Battery. A few months ago I began using it again, and it seems fine, but I agree that the PineTime's lack of a "power-off"-Function is annoying.

Wasp-os can sadly not help with this, as this is a hardware issue :(

**Comment by jtolio** [comment]
I actually have a Colmi P8, which had a "shutdown" option in the settings before I reflashed it.
